### ISSUE TITLE
We need more visitor fobs, or to find the existing visitor fobs, or perhaps secret option c

### ISSUE THREAD

**Issue by Robotorium** [DoESLiverpool/somebody-should#2082]
We have a visitor in the events space who needs a fob, and I've had to lend them mine because the visitor fobs box is empty, and no fobs are visible roaming wild and free around the space.

Cheers ears!

**Comment by zarino** [comment]
I noticed last week, when a user of the event space asked for a fob, that there was only one left in the box (just a blue fob, on its own, no keyring or anything). I’m fairly sure I saw it back in the box the day after, so they must have returned it. But I’m not surprised if it’s gone missing since.

**Comment by huffeec** [comment]
I've written three temporary fobs visitor_temp1, visitor_temp2 and visitor_temp3 so we have some for now. They are on borrowed Makefest lanyards to make them harder to lose. They are in the quality street box.

I'll leave this issue open because the lost fobs need to be removed from the system but I'm not clear which these are.

**Comment by DoESsean** [comment]
I've deleted 7 temp/kitchen/visitor passes, left the three that Chris has
created.
